setup.py

```diff
@@ -29,7 +29,7 @@
     "tqdm",
     "pytz",
     "web3==5.28.0",
-    "cryptography==3.3.2",
+    "cryptography",
```

Comment:
there were some discussions about cryptography , especially on MacOS
maybe it's better if we force it to a specific version

Reply:
The issues before were because it had been forced to a specific (older) version. If not forcing solves the issue, keep like that:)

Reply:
More detail; The issue in question is : https://github.com/oceanprotocol/ocean.py/issues/1034

And @jfdelgad had found that removing the constraint.

TBH this really should be its own PR, to make it easy for people to track.

Give me a minute, I'll do that...

Reply:
OK, done. @jfdelgad please merge main into your branch, and then this change will go away:)

Reply:
I'm not a big fan of wild dependency  (ie: with no fixed version). That can lead to different scenarios and bugs, very hard to reproduce.

Therefore I recommend "  cryptography==38.0.1"  (https://github.com/oceanprotocol/ocean.py/actions/runs/3339736289/jobs/5529041238#step:6:30)